docs/tools/CLI/build_profiles.md

```diff
@@ -1,42 +1,42 @@
 <h2 id="build-profiles">Build profiles</h2>
 
-Arm Mbed OS 5 supports three primary build profiles: *develop*, *debug* and *release*. The Online Compiler uses the *develop* profile. When building from Arm Mbed CLI, you can select a profile by adding the `--profile <profile>` flag. You can specify custom user-defined profiles by giving the path to the profile.
+Arm Mbed OS 5 defines three collections of toolchain flags used during the build. These are __build profiles__. These are collections of toolchain flags used during the build. The three build profiles are *develop*, *debug* and *release*. The Mbed Online Compiler uses the *develop* build profile. When building from Arm Mbed CLI, you can select the build configuration by adding the `--profile <`__build profile__ `name>` flag. You can also specify custom- or user-defined configurations by giving the path to the JSON file defining the build configuration.
```

Comment:
Remove redundant definition and "build configuration" or "configuration" -> "build profile":
```suggestion
Arm Mbed OS 5 defines three collections of toolchain flags used during the build. These are __build profiles__. The three build profiles are *develop*, *debug* and *release*. The Mbed Online Compiler uses the *develop* build profile. When building from Arm Mbed CLI, you can select the build profile by adding the `--profile <`__build profile__ `name>` flag. You can also specify custom- or user-defined build profiles by giving the path to the JSON file defining the build profile.
```

Further, I'm not sure I like
> by adding the `--profile <`__build profile__ `name>` flag.

I would instead write something like 
> you may select the build profile by passing your desired build profile, by name or path, to the `--profile` argument.

Reply:
Apparently, I don't have permissions to push to this repository when using the "commit this suggestion" button, but I can make the changes directly?